engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/views/content-hosts.html

```diff
@@ -5,6 +5,9 @@ <h2 translate>Content Hosts</h2>
   </div>
 
   <div data-block="list-actions">
+    <a href="/hosts/content_hosts.csv?{{csvQuery}}" class="btn btn-default">
+      <span translate>Export</span>
+    </a>
```

Comment:
seeing a small issue with using this.  It causes the URL to change which messes up things such as:
a) clicking export a 2nd time doesn't work
b) clicking 'search' does cause a 2nd download

adding `target="_self"`  seems to fix it.  @waldenraines is that appropriate here?